mythril/laser/ethereum/util.py

```diff
@@ -89,10 +89,10 @@ def get_concrete_int(item):
 
 
 def concrete_int_from_bytes(_bytes, start_index):
-
-    # logging.debug("-- concrete_int_from_bytes: " + str(_bytes[start_index:start_index+32]))
+    _bytes = [
+        _byte.as_long() if type(_byte) == BitVecNumRef else _byte for _byte in _bytes
+    ]
     b = _bytes[start_index : start_index + 32]
```

Comment:
While you are busy can you make the variable names in this function a bit more verbose. integer_bytes seems more clear than `b` And the int.from_bytes can directly be returned vs creating a variable val